scale/job/models.py

```diff
@@ -392,6 +392,11 @@ def get_details(self, job_id):
         else:
             job.execution = None
 
+        configuration = job.get_job_configuration()
+        manifest = SeedManifest(job.job_type.manifest, do_validate=False)
```

Comment:
almost there. we should get the manifest for the job type revision, not necessarily the latest one. Probably use the get_job_interface method.

Reply:
Also we should add a test that DB_PASS is removed